package.json

```diff
@@ -55,7 +55,7 @@
   "devDependencies": {
     "@types/node": "^18.7.20",
     "esbuild-register": "^3.3.3",
-    "msw": "^1.2.1",
+    "msw": "^2.0.0",
```

Comment:
probably doesn’t matter since we’re running this in the console anyway, but this should probably be 2.0.2